cmd/kube-proxy/app/server.go

```diff
@@ -477,6 +477,11 @@ func NewProxyServer(config *componentconfig.KubeProxyConfiguration, cleanupAndEx
 			return nil, fmt.Errorf("unable to read IPTables MasqueradeBit from config")
 		}
 
+		nodeIP, err := getNodeIP(client, hostname)
+		if err != nil {
+			return nil, fmt.Errorf("unable to fetch node IP: %v", err)
```

Comment:
@MikeSpreitzer  I know. With this change, if someone is running a standalone kube-proxy. It will fail. So I am wondering if it is reasonable to grab bind-address for this.